pkg/machinery/config/provider.go

```diff
@@ -171,7 +171,7 @@ type Route interface {
 // Time defines the requirements for a config that pertains to time related
 // options.
 type Time interface {
-	Enabled() bool
+	Disabled() bool
```

Comment:
strictly speaking, we didn't have to change the interface, but this way it's more consistent ;)

Reply:
yeah, seemed like it could cause too much confusion if it didn't match.